frontend/src/entry/model/releases/EditableRelease.tsx

```diff
@@ -214,6 +216,7 @@ export default function EditableRelease({ release, isEdit, onIsEditChange }: Edi
         onFilesChange={(value) => setFiles(value)}
         onFilesMetadataChange={(value) => setFilesMetadata(value)}
         onImageListChange={(value) => setImageList(value)}
+        setRegistryError={(value) => setRegistryError(value)}
```

Comment:
As this function is being used in a `useEffect` it must be stable. Currently the function is recreated on every render, which will cause an infinite loop. To prevent this I would add the following logic to this file (I have suggested renaming in other comments, but have left the names untouched in the below snippet):
```
// Change this line to
setRegistryError={handleRegistryError}

// Add the following to this component to make this function stable
const handleRegistryError = useCallback((value: boolean) => setRegistryError(value), [])
```